src/db_adapters/accounts.rs

```diff
@@ -141,3 +141,62 @@ async fn remove_accounts(
         }
     }
 }
+
+pub(crate) async fn store_accounts_from_genesis(near_config: near_indexer::NearConfig) {
+    info!(
+        target: crate::INDEXER_FOR_EXPLORER,
+        "Adding/updating accounts from genesis..."
+    );
+    let pool = crate::models::establish_connection();
+
+    let accounts_models: Vec<models::accounts::Account> = near_config
+        .genesis
+        .records
+        .0
+        .iter()
+        .filter_map(|record| {
+            if let near_indexer::near_primitives::state_record::StateRecord::Account {
+                account_id,
+                ..
+            } = record
+            {
+                Some(models::accounts::Account::new(account_id.to_string(), None))
+            } else {
+                None
+            }
+        })
+        .collect();
+
+    loop {
+        match diesel::insert_into(schema::accounts::table)
+            .values(accounts_models.clone())
+            .on_conflict(schema::accounts::dsl::account_id)
+            .do_update()
+            .set((
+                schema::accounts::dsl::created_by_receipt_id
+                    .eq(excluded(schema::accounts::dsl::created_by_receipt_id)),
+                schema::accounts::dsl::deleted_by_receipt_id
+                    .eq(excluded(schema::accounts::dsl::deleted_by_receipt_id)),
+            ))
```

Comment:
I think we should do nothing with the account that was created in response to some execution outcome (which happens after genesis). We may even want to only dump genesis if the database is completely blank since it does not make a lot of sense to mix some stale state with the fresh genesis dump :thinking: